scripts/13-ocp-iam-users.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Use this script to grant access to IBM Cloud users to ROKS cluster
+# Update the IAM_USERS list below with the list of user email addresses
+# then run the script ./13-ocp-iam-users.sh
+
+set -euo pipefail
+
+TOOLKIT_GROUP=${TOOLKIT_GROUP:-ibm-toolkit-users}
+IAM_TOOLKIT_GROUP=${IAM_TOOLKIT_GROUP:-ibm-toolkit-iam-users}
+IAM_PREFIX="IAM#"
+IAM_USERS="user1@email.com \
```

Comment:
You can have this as comments as example at the top, and make it a requirement to provide list of user IDs, and optional to provide a prefix, or don't provide one.

You can use variables like "NEW_USERS" and optional "NEW_USER_PREFIX"